src/constants/shards.ts

```diff
@@ -1,72 +1,72 @@
+import { ZoneData } from './zones.js';
+
+export enum Shard {
+    Cyprus = '0x0',
+    Cyprus1 = '0x00',
+    Cyprus2 = '0x01',
+    Cyprus3 = '0x02',
+    Paxos = '0x1',
+    Paxos1 = '0x10',
+    Paxos2 = '0x11',
+    Paxos3 = '0x12',
+    Hydra = '0x2',
+    Hydra1 = '0x20',
+    Hydra2 = '0x21',
+    Hydra3 = '0x22',
+    Prime = '0x',
+}
+
+function fromBytes(shard: string): Shard {
```

Comment:
I know that this method is not exported but I believe the function naming could be more clear by changing it to `zoneFromBytes`, `deriveZoneFromBytes` or something along those lines

Reply:
@rileystephens28 I thought about that but decided fromBytes made more sense given the context. I can change it to have the context in the name though.